shared/utils/limit.go

```diff
@@ -22,7 +20,7 @@ func FindMemoryLimit(x, y string) int {
 	if x != "" {
 		bytes, err := ParseMemoryLimit(x)
 		if err != nil {
-			log.WithError(err).Errorf("Failed to parse global RAM limit %s", x)
+			log.With("error", err).Error("Failed to parse global RAM limit", "limit", x)
```

Comment:
```suggestion
			log.With("error", err).Error("Failed to parse global RAM limit: ", x)
```

Reply:
We need to specify it. Otherwise, we get an error just like the one stated here: https://tip.golang.org/src/cmd/vendor/golang.org/x/tools/go/analysis/passes/slog/doc.go

Reply:
OK - A very strange design decision on their end in my opinion. Thanks for the clarification :)